### ISSUE TITLE
Bug when resizing window from "System Setup" tab

### ISSUE THREAD

**Issue by luckman212** [knurling/ServiceStation#35]
- 2020.5
- 10.15.4

This is a weird one! Only happens when **System Setup** tab is selected... while vertically resizing the window, it also steadily drifts to the right. If you continue dragging the mouse up and down, the window will eventually end up completely offscreen.

A video shows it best...

![ghost_window 2020-05-13 22_26_22](https://user-images.githubusercontent.com/1992842/81885651-cc015c00-9568-11ea-9c59-ee9577de254f.gif)

**Comment by pkamb** [comment]
Yeah - this is a funny one that I haven't been able to track down yet. On the list.

Auto Layout issue that for some reason is moving the window 1px at a time. Probably due to the locked horizontal sizing.
